src/storage/mvcc/reader/forward_seeker.rs

```diff
@@ -136,12 +136,13 @@ pub struct ForwardSeeker<S: Snapshot> {
     /// `default cursor` is lazy created only when it's needed.
     default_cursor: Option<Cursor<S::Iter>>,
 
-    statistics: Statistics,
+    /// Is iteration started
+    is_started: bool,
```

Comment:
maybe.. is_initialized is better?

Reply:
It just marks whether `read_next` is already called once. I think `is_started` is just ok here.